### PR TITLE
feature: use load_model() function

### PR DESCRIPTION
**Pull request by YuriCat**
`load_model()` is more natural than `get_model()`.

Especially for ONNX models, the model can be loaded with just the `model_path`.
Therefore, it makes more sense to have the `model_path` as the first argument, as in build_agent(), but I'm not sure.

### CODE REVIEW

**Comment by ikki407** [comment]
IMHO, it's natural to pass the model first and then the weights (`load_model(model, model_path)`).
How about using a model as an argument?

E.g.
```python
model = env.net()
model = load_model(model_path, model)
# or
model = load_model(env.net(), model_path)
```

**Comment by YuriCat** [comment]
I was worried that unnecessary `torch` would be imported when we loaded ONNX models.
However, as long as the neural net is defined in the environment file, torch is already imported when an environment instance is created.

**Comment by ikki407** [comment]
Ah, I got it!!

**Comment by ikki407** [comment]
So, if the definition of the neural network is separated from the environment module, `load_model()` can return an ONNX model without importing PyTorch when the extension is `.onnx`.

**Comment by YuriCat** [comment]
That's right. This torch lazy import will be written by whoever needs it, and I think your idea is cool.

File: handyrl/evaluation.py [hunk 1]
```diff
@@ -277,10 +277,9 @@ def network_match_acception(n, env_args, num_agents, port):
     return agents_list
 
 
-def get_model(env, model_path):
+def load_model(model_path, model):
```
Comment (ikki407) [inline handyrl/evaluation.py:280]: 👍

**Comment by ikki407** [comment]
LGTM

**Pull request merged**